tests/v2/schemaTest/schema_test.go

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import (
+	"fmt"
+	"github.com/devfile/library/tests/v2/utils"
+	"strings"
+	"testing"
+
+	devfilepkg "github.com/devfile/library/pkg/devfile"
+)
+
+const (
+	jsonDir      = "../json/"
+	logErrorOnly = false
+)
+
+// Test_Schema: Feeds a variety of devfiles to th parse to check it parses and validates the devfile correctly.
+func Test_Schema(t *testing.T) {
+
+	passTests := 0
+	totalTests := 0
+	skippedTests := 0
+
+	// Get all of the test defined in teh json file
+	testsToRun, failure := utils.GetAllTests(jsonDir)
+	if failure {
+		t.Error(utils.LogErrorMessage(fmt.Sprintf("At least one error occurred reading tests from %s, see log for details", jsonDir)))
+		totalTests++
+	}
+
+	if len(testsToRun) == 0 {
+		t.Fatal(utils.LogErrorMessage("No tests were found!"))
+	}
+
+	// Run each test
+	for _, testToRun := range testsToRun {
+
+		totalTests++
+
+		// Test may be disabled pending changes to the schema
+		if testToRun.Disabled {
+			t.Log(utils.LogInfoMessage(fmt.Sprintf("SKIP : %s", testToRun.FileName)))
+			skippedTests++
```

Comment:
Please mention about the `Disabled` property in README

Reply:
Done.